vmdb/app/presenters/menu/section.rb

```diff
@@ -8,6 +8,15 @@ def features
       Array(items).collect(&:feature).compact
     end
 
+    def visible?(userid)
+      store = Vmdb::PermissionStores.instance
+      tab   = Menu::Manager.tab_features_by_id(id)
+      auth  = store.can?(id) && User.current_user.role_allows_any?(:identifiers => tab)
```

Comment:
@jrafanie : the problem is the part before the `&&` -- the store does not know about the custom menu section and items.

I started putting together some specs https://github.com/ManageIQ/manageiq/pulls/martinpovolny but I cannot capture the behavior needed here, because the permission is checked firts in the partial.

So I actually need to take this `visible?` from here w/o the `PermissionStores` part and add it to the Menu::Manager and then I can write a spec to capture the behavior that is broken by this PR. Will do that....